src/pages/account/index.test.tsx

```diff
@@ -13,6 +13,10 @@ jest.mock('../../features/account/components/AccountInfo', () => ({
   AccountInfo: () => <div />,
 }));
 
+jest.mock('../../features/account/AccountTEEndorsement', () => ({
```

Comment:
minor : to test if mocked endorse component is present on index page.

Reply:
right, i just override the the whole endorsement components for easier test, but may be it is better to keep it for test if it present or not